aspnetcore/blazor/fundamentals/static-files.md

```diff
@@ -203,6 +187,22 @@ The preceding code results in the following import map:
 }
 ```
 
+<!-- HOLD
```

Comment:
When is this going to be available?

Reply:
Tomorrow morning I'll put up the PR, but it might merge early next week if I hit any snags.

Reply:
BTW... I'm going to look this PR over one more time in the morning before merging. The old dino 👴 has gone to 🛌 .